source/tutorials/moving-pictures.rst

```diff
@@ -327,6 +327,33 @@ Finally, ordination is a popular approach for exploring microbial community comp
 .. question::
     What differences do you observe between the unweighted UniFrac and Bray-Curtis PCoA plots?
 
+Alpha rarefaction plotting
+--------------------------
+
+In this section we'll explore alpha diversity as a function of sampling depth using the ``qiime diversity alpha-rarefaction`` visualizer. This visualizer computes one or more alpha diversity metrics at multiple sampling depths, in steps between 1 (optionally controlled with ``--p-min-depth``) and the value provided as ``--p-max-depth``. At each sampling depth step, 10 (optionally controlled with ``--p-iterations``) rarefied tables will be generated, and the diversity metrics will be computed for all samples in the tables. Average diversity values will be plotted for each sample at each even sampling depth, and samples can be grouped based on metadata in the resulting visualization if sample metadata is provided.
+
+.. command-block::
+
+   qiime diversity alpha-rarefaction \
+     --i-table table.qza \
+     --i-phylogeny rooted-tree.qza \
+     --p-max-depth 4000 \
+     --m-metadata-file sample-metadata.tsv \
+     --o-visualization alpha-rarefaction.qzv
+
+The visualization will have two plots. The top plot is an alpha rarefaction plot, and is primarily used to determine if the richness of the samples has been fully observed or sequenced. If the lines in the plot appear to "level out" (i.e., approach a slope of zero) at some sampling depth along the x-axis, that suggests that collecting additional sequences beyond that sampling depth would not be likely to result in the observation of additional features. If the lines in a plot don't level out, this may be because the richness of the samples hasn't been fully observed yet (because enough sequences weren't collected), or it could be an indicator that a lot of sequencing error remains in the data (which is being mistaken for novel diversity).
+
+The bottom plot in this visualization is important when grouping samples by metadata. It illustrates the number of samples that are in each group at each even sampling depth. If a given sampling depth ``d`` is larger than the total frequency of a sample ``s`` (i.e., the number of sequences that were obtained for sample ``s``), it is not possible to compute the diversity metric for sample ``s`` at sampling depth ``d``. If many of the samples in a group have lower total frequencies than ``d``, the average diversity presented for that group at ``d`` in the top plot will be unreliable because it will have been computed on relatively few samples. When grouping samples by metadata, it is therefore essential to look at the bottom plot to ensure that the data presented in the top plot is reliable.
```

Comment:
> It illustrates the number of samples that are in each group at each even sampling depth

Maybe change to "It illustrates the number of samples that remain in each group when the feature table is evenly rarefied at each sampling depth"?